src/Components/Admins/Form/index.js

```diff
@@ -100,6 +104,10 @@ const Form = () => {
     setShowErrorMessage(false);
   };
 
+  if (isLoading) {
+    return <IsLoading />;
+  }
+
```

Comment:
I don't think curly braces are needed for this case
```suggestion
  if (isLoading) return <IsLoading />;
```